frontend/lib/about/dev_settings_view.dart

```diff
@@ -171,71 +164,8 @@ class DevSettingsView extends StatelessWidget {
     );
   }
 
-  Future<void> _activateCard(BuildContext context, String base64qrcode) async {
```

Comment:
Why is all of this no longer needed?

Reply:
Because it's moved to the deep link Activation. Bow I just forward from de settings to deep link activation